packages/eds-core-react/src/components/SideSheet/SideSheet.stories.tsx

```diff
@@ -43,3 +78,200 @@ export const Default: Story<SideSheetProps> = (args) => {
     </div>
   )
 }
+
+export const Placement: Story<SideSheetProps> = () => {
+  const cellValues = toCellValues(data, columns)
+  const [toggle, setToggle] = useState(false)
```

Comment:
Rename to "open" so its the same as in the other examples?
Suggest default state to be open so its clearer to users that this is about side sheet.